manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -613,3 +613,8 @@ properties:
 
   garden:
     log_level: error
+  racoon:
+    certificate_authority_cert: (( grab secrets.bosh_ca_cert ))
+    certificate_authority_private_key: (( grab secrets.bosh_ca_key ))
```

Comment:
I don't feel comfortable of distributing our CA private key on the VMs. It even ends up in a [world readable file](https://github.com/SAP/ipsec-release/blob/1d27c9115adc838d0cfac402a6615d0dbf16a912/jobs/racoon/templates/etc/racoon/certs/ca_pkey.pem.erb).
I wish we could generate the certificates upfront and provide them via properties. That would require forking this release.
In the meantime, we could create a dedicated CA for IPsec. At least if the key leaks, only IPSec would be affected...